packages/tocco-ui-showcase/src/components/ShowCaseApp.js

```diff
@@ -38,6 +39,7 @@ export default class ShowCaseApp extends React.Component {
       <ThemeProvider theme={ToccoTheme}>
         <IntlProvider locale={this.state.locale}>
           <div className="show-case-app tocco-ui-theme">
+            <InjectFontRoboto theme={ToccoTheme}/>
```

Comment:
Is there a more generic way to inject the required font files? maybe by creating a higher order component for ThemeProvider, do everything there, and just use this extended ThemeProvider everywhere instead of the original one.
Inserting this `InjectFontRoboto` tag everywhere is something that tends to be forgotten easily. Furthermore, what happens if we decide to use a different font? Right now we would have to replace this element everywhere.

I don't mind whether we refactor this in this pull request or in a following change, but I think we should rethink this approach. What do you think?

Reply:
I agree, this solution is not flexible enough and adds unnecessarily tags. It does not provide options to add a dev key for paid font service or font files. 

Adding two times the component itself seems okay to me. I am sure better solution would exist. I'd like to discuss this issue and I'm happy about some advice.

@MethodenMann Is it okay to add a task to your icebox?

Reply:
Pivotal Story ID 158457342